### DISCUSSION TITLE
Join us on Discord

### DISCUSSION THREAD

**Discussion by aduros** [aduros/wasm4#70] [General]
Discord invite link: https://discord.com/invite/7teRdHdbYk

Come on by and say hello!

**Comment by peterhellberg** [comment]
@aduros The invite link looks to be invalid now.

**Reply by aduros** [reply]
Oops, I guess discord invite links expire by default... here's one set to never expire https://discord.gg/NFcWMxh4a6

**Reply by amirouche** [reply]
Here is a link that works taken from the website https://discord.com/invite/7teRdHdbYk
